elkodon/src/port/publisher.rs

```diff
@@ -17,8 +17,18 @@
 //!     .unable_to_deliver_strategy(UnableToDeliverStrategy::DiscardSample)
 //!     .create()?;
 //!
-//! // loan some memory and send it
+//! // loan some initialized memory and send it
```

Comment:
Could you please mention that this requires `Default` to be implemented and that this is used to initialize the memory.